src/components/OcTextarea.vue

```diff
@@ -212,7 +212,7 @@ export default {
     @extend .uk-flex-middle;
     @extend .oc-mt-xs;
 
-    min-height: $global-font-size * 1.5;
+    min-height: $oc-font-size-default * 1.5;
```

Comment:
Why are we using a SASS variable here?

Reply:
i had problems to mix css variables and sass calculations